Jenkinsfile-pr

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env groovy
+
+def lib = evaluate readTrusted('./jenkins.groovy')
+
+pipeline {
+    agent {
+        node {
+            label 'strimzi-pr'
+        }
+    }
+    parameters {
+        //execute all tests as default for each new Pull request
+        string(name: 'JUNIT_TAG', defaultValue: '', description: 'maven parameter for executing specific tests')
+    }
+    options {
+        timeout(time: 7, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+    environment {
+        DOCKER_ORG="strimzi"
+        DOCKER_REGISTRY="docker.io"
+        DOCKER_TAG="latest"
+        MVN_ARGS="-DskipTests"  // Remove this when unit tests will be stabilized TODO jstejska
```

Comment:
If you're not intending to fix this as part of this PR then it's better to open an issue than to leave a TODO in the code.

Reply:
I agree, I guess it's env issue...some of the systemtes (MM for example) failing as well